Cargo.toml

```diff
@@ -3,6 +3,7 @@ repository = "https://github.com/w3f/ring-vrf/"
 
 
 [workspace]
+resolver = "2"
```

Comment:
Is this just for dev-dependencies or something else?  https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2

Reply:
Is just to get rid of this warning
```
warning: some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
 ```